### DISCUSSION TITLE
your settings.xml is 4 years old! / magic draw uml

### DISCUSSION THREAD

**Discussion by littlenici** [eclipse-sirius/sirius-components#3154] [Q&A] [answered]
Hi, 
I downloaded all packages from sirius-web on Github. Now I realized that I cannot build the project since the settings.xml in the root path is outdated - 4 years old and the references to the repository and the github-sirius-emfjson and github-flow are not working:
-> https://maven.pkg.github.com/eclipse-sirius/sirius-emf-json
-> https://maven.pkg.github.com/ObeoNetwork/Flow-Designer

https://github.com/eclipse-sirius/sirius-web#build

Your build discription includes no word about the outdatet settings.xml.
How would you advise to build the sirius-web? 

And another question do you support the magic draw *.uml files?
Can I import them to sirius-web? if not, how would you advise me, to integrate them to sirius-web?

**Comment by AxelRICHARD** [comment, marked as answer]
Hello,
The `settings.xml` located at the root of the repository is only here for the continuous integration with github actions. It is not outdated.
You should not use it while building locally, but rather use your own as described [here](https://github.com/eclipse-sirius/sirius-web?tab=readme-ov-file#github-access-token).
The sample application provided inside the Sirius Web repository does not handle UML models.
But you can create your own Sirius Web-based application to handle UML models, or you can also visit https://github.com/PapyrusSirius/papyrus-web which is a Sirius Web-based application for UML models.
Hope it will help you.
Regards,

**Comment by littlenici** [comment]
hi Axel, thank you for your fast reply, this helped me.
